constants.xml

```diff
@@ -91,6 +91,7 @@
         <GET_INITIAL_DS_FROM_REPO>false</GET_INITIAL_DS_FROM_REPO>
         <UPGRADE_HOST_ACCOUNT>Zilliqa</UPGRADE_HOST_ACCOUNT>
         <UPGRADE_HOST_REPO>Zilliqa</UPGRADE_HOST_REPO>
+        <SEND_RESPONSE_FOR_LAZY_PUSH>true</SEND_RESPONSE_FOR_LAZY_PUSH>
```

Comment:
Seems more and more gossip-related options in constant file. Is it possible to group them?

Reply:
created issue - https://github.com/Zilliqa/Issues/issues/339